OpenSim/Simulation/Model/OffsetFrame.h

```diff
@@ -90,7 +95,7 @@ class  OffsetFrame : public C {
     offset property of this OffsetFrame.
      
     @param[in] parent   The parent reference frame.
-    @param[in] offset The offset transform between this frame and its parent
+    @param[in] offset   The offset transform between this frame and its parent
     */
     OffsetFrame(const C& parent, const SimTK::Transform& offset);
```

Comment:
This comment doesn't require any action..just ruminating...it seems like with C++11 we can replace such const pass-by-reference's (in this case the object eventually gets copied I think) with pass-by-value, exploiting move constructors. See http://clang.llvm.org/extra/PassByValueTransform.html.

Anyway, I'm not suggesting any changes here.